### PR TITLE
Som devel 20 issue #45

### PR DESCRIPTION
**Pull request by sorgom**

### CODE REVIEW

File: application/components/BAS/README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 **I_Array.h**
 ```
-defintion of interface I_Array:
+definition of interfaces I_Array / I_SortableArray
 enables:
```
Comment (sombeam) [inline application/components/BAS/README.md]: move this to I_SortableArray

File: testing/testenv/testlib/MutableArray.h [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+    NOCOPY(MutableArray)
+
+private:
+    constexpr static auto DIM = sizeof(C);
+    using Segment = BYTE[DIM];
+    Segment mData[CAP];
+    size_t mSize;
+};
+
```
Comment (sombeam) [inline testing/testenv/testlib/MutableArray.h]: reduce empty lines

**Review by sombeam — CHANGES_REQUESTED** [review]

File: application/components/BAS/README.md (hunk 1 above)
Comment (sorgom) [reply application/components/BAS/README.md]: done

File: testing/testenv/testlib/MutableArray.h (hunk 2 above)
Comment (sorgom) [reply testing/testenv/testlib/MutableArray.h]: done

File: testing/testenv/testlib/README.md [hunk 3]
```diff
@@ -30,8 +30,22 @@ ProjData generator
...
+
     additional features
-    -   set signal type for signal data
+    -   set SIG type for signal data
+    -   set LCR type for level crossing data
+```
+
+**MutableArray.h**
+```
+array types for test issues
```
Comment (sombeam) [inline testing/testenv/testlib/README.md:42]: "array types for test issues"

then it should be called TestArrays.h

**Review by sombeam — CHANGES_REQUESTED** [review]

File: testing/testenv/testlib/README.md (hunk 3 above)
Comment (sorgom) [reply testing/testenv/testlib/README.md:42]: OK, done

**Review by sombeam — APPROVED** [review]

**Pull request merged**